server/channels/app/export_test.go

```diff
@@ -804,3 +804,369 @@ func TestExportArchivedChannels(t *testing.T) {
 	}
 	require.True(t, found, "archived channel not found after import")
 }
+
+func TestExportRoles(t *testing.T) {
```

Comment:
Most of the PR is these tests really. It's a bit verbose but I want to make sure the import/export process is sane.